Source/Scene/QuadtreePrimitive.js

```diff
@@ -247,6 +283,7 @@ define([
         }
 
         var i;
+        var j;
```

Comment:
This comment wasn't updated, but a billboard will stop updating when the maximum level has been reached.